Makefile

```diff
@@ -1,103 +1,141 @@
+.PHONY: help install-python install-js install-test install \
+		update-npm-requirements javascript sass \
+		extract_translations compile_translations generate_dummy_translations validate_translations \
+		detect_changed_source_translations pull_translations push_translations check_translations_up_to_date \
+		quality test-python render-templates test-js test-js-debug test test-acceptance test-a11y test-sandbox \
+		install-osx-requirements
+
+.DEFAULT_GOAL := help
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
```

Comment:
try switching `.` to `openassessment/` in these lines, so that you don't clean things out of the virtualenv.